README.md

```diff
@@ -1,32 +1,44 @@
 Node Apps Toolkit
 ===
 
-A collection of helpers and utilities for creating NodeJS Contentful Apps
+The Node Apps Toolkit is a growing collection of helpers, and utilities, for creating NodeJS Contentful Apps.
 
 ## Getting started
  
-You can install this library via
+You can install this library with npm or yarn
  
 ```
 npm install --save @contentful/node-apps-toolkit
+or
+yarn add @contentful/node-apps-toolkit
 ```
 
-and include it in your code like
+## Features
 
+### getManagementToken
+NodeJS Contentful Apps need a management token to interact with Contentful's APIs.
+Creating a management token requires a key pair to be regsitered for the app, follow [this link](http://contentful./developers/docs/references/content-management-api/#/reference/app-keys/app-keys) for more information on key pairs.
+Once a key pair is registered the getManagementToken function can be used to generate a valid token.
+
+This example demonstrates how to import and use `getManagementToken`
 ```js
-const {getManagementToken} = require('@contentful/node-apps-toolkit');
-const {appInstallationId, spaceId, privateKey} = require('./some-constants');
+import { getManagementToken } from '@contentful/node-apps-toolkit';
+import { appInstallationId, spaceId, privateKey } from './some-constants';
```

Comment:
I would leave this with `require`s: the toolkit is meant to work on barebone Node without necessarily using modules, Typescript, Babel or transpiling logic. `import` may give the false impression we don't support CommonJS. WDYT?